spec/lib/sidekiq/form526_backup_submission_process/submit_spec.rb

```diff
@@ -84,13 +84,30 @@
                 expect(jid).to eq(jid_from_jobs)
                 described_class.drain
                 expect(jid).not_to be_empty
+
+                # After 526 succeeds ancillary form go through their backup submission process
+
+                # Form 4142 Backup Submission Process
+                expect(submission.form['form4142']).not_to be(nil)
+                form4142_processor = DecisionReviewV1::Processor::Form4142Processor.new(
+                  form_data: submission.form['form4142'], submission_id: submission.id
+                )
+                request_body = form4142_processor.request_body
+                metadata_hash = JSON.parse(request_body['metadata'])
+                form4142_received_date = metadata_hash['receiveDt'].to_date
+                expect(submission.created_at.to_date).to eq(form4142_received_date)
```

Comment:
Using the `DateTime` class, I suppose it would be reasonable to expect `submission.created_at` to be within a second of `metadata_hash['receiveDt']`, if more precision was desired.